web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElement.tsx

```diff
@@ -177,24 +182,35 @@ export const AnytimeActionTaxClearanceCertificateElement = (props: Props): React
                   <Heading level={1}>{props.anytimeAction.name}</Heading>
                 </div>
               </div>
-              <HorizontalStepper
-                steps={stateTaxClearanceCertificateSteps}
-                currentStep={stepIndex}
-                onStepClicked={(step: number): void => {
-                  if (step === 2 && stepIndex === 1) {
-                    saveTaxClearanceCertificateData();
-                  }
-                  setStepIndex(step);
-                }}
-              />
-              {stepIndex === 0 && <TaxClearanceStepOne setStepIndex={setStepIndex} />}
-              {stepIndex === 1 && (
-                <TaxClearanceStepTwo
-                  setStepIndex={setStepIndex}
-                  saveTaxClearanceCertificateData={saveTaxClearanceCertificateData}
-                />
+              {certificatePdfArray && <TaxClearanceDownload />}
+
+              {!certificatePdfArray && (
+                <>
+                  <HorizontalStepper
+                    steps={stateTaxClearanceCertificateSteps}
+                    currentStep={stepIndex}
+                    onStepClicked={(step: number): void => {
+                      if (step === 2 && stepIndex === 1) {
+                        saveTaxClearanceCertificateData();
+                      }
+                      setStepIndex(step);
+                    }}
+                  />
+                  {stepIndex === 0 && <TaxClearanceStepOne setStepIndex={setStepIndex} />}
+                  {stepIndex === 1 && (
+                    <TaxClearanceStepTwo
+                      setStepIndex={setStepIndex}
+                      saveTaxClearanceCertificateData={saveTaxClearanceCertificateData}
+                    />
+                  )}
+                  {stepIndex === 2 && (
+                    <TaxClearanceStepThree
+                      setStepIndex={setStepIndex}
+                      setCertificatePdfArray={setCertificatePdfArray}
+                    />
+                  )}
+                </>
```

Comment:
[dust] I would probably extract this into a function that returns a TSX element.